library/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/SessionReplayConfiguration.kt

```diff
@@ -17,7 +18,8 @@ data class SessionReplayConfiguration
 internal constructor(
     internal val endpointUrl: String,
     internal val privacy: SessionReplayPrivacy,
-    internal val extensionSupport: ExtensionSupport
+    internal val extensionSupport: ExtensionSupport,
+    internal val samplingRate: Float
```

Comment:
for the naming consistency

```suggestion
    internal val sampleRate: Float
```